### ISSUE TITLE
Scrolling resets when the markup changes

### ISSUE THREAD

**Issue by lenski88** [ticlo/rc-dock#237]
If you drag tabs, neighboring tabs may lose the scroll value. This can be seen in the examples. For example, [.](https://ticlo.github.io/rc-dock/examples/#controlled-layout) The jsx tab has scrolling, and if you move a new tab to it, creating two panels, the scrolling will reset, or if you move the jsx tab itself to another place in the markup

**Comment by rinick** [comment]
have you tried cached tab?
https://ticlo.github.io/rc-dock/examples/#tab-cache

**Comment by lenski88** [comment]
Yes, I tried. are you saying that jsxTab is not cached in your examples? I had a problem with the scroll reset when resizing the size of the panels. The cached property was set to true, but the problem was that it generated a new id for the tab every time. Do you think there's still this problem?
